tests/test_opensubtitles.py

```diff
@@ -6,8 +6,8 @@
 import pytest
 from vcr import VCR
 
-from subliminal.providers.opensubtitles import OpenSubtitlesProvider, OpenSubtitlesSubtitle
-
+from subliminal.exceptions import ConfigurationError
+from subliminal.providers.opensubtitles import OpenSubtitlesProvider, OpenSubtitlesSubtitle, OpenSubtitlesError
```

Comment:
from subliminal.providers.opensubtitles import OpenSubtitlesError, OpenSubtitlesProvider, OpenSubtitlesSubtitle

I try to keep imports alphabetically ordered (with classes before functions).